### PR TITLE
CDAP-6570 Fixed bug in ActionContext to setup arguments correctly.

### PR DESCRIPTION
**Pull request by sagarkapare**
Fixed bug in the ActionContext.
Build is running here - http://builds.cask.co/browse/CDAP-DUT4447-1

### CODE REVIEW

**Comment by sagarkapare** [comment]
@albertshau @gsps1 Please take a look. Thanks!

File: cdap-app-templates/cdap-etl/cdap-data-pipeline/src/test/java/co/cask/cdap/datapipeline/DataPipelineTest.java [hunk 1]
```diff
@@ -138,6 +142,28 @@ public void testPipelineWithAllActions() throws Exception {
...
+
+    tokenDetail = workflowManager.getToken(runId, WorkflowToken.Scope.USER, action2RowKey + action2ColumnKey);
+    validateToken(tokenDetail, action2RowKey + action2ColumnKey, action2Value);
+
+    tokenDetail = workflowManager.getToken(runId, WorkflowToken.Scope.USER, action3RowKey + action3ColumnKey);
+    validateToken(tokenDetail, action3RowKey + action3ColumnKey, action3Value);
+  }
+
+  private void validateToken(WorkflowTokenDetail tokenDetail, String key, String value) {
+    List<WorkflowTokenDetail.NodeValueDetail> nodeValueDetails = tokenDetail.getTokenData().get(key);
```
Comment (gsps1) [inline cdap-app-templates/cdap-etl/cdap-data-pipeline/src/test/java/co/cask/cdap/datapipeline/DataPipelineTest.java:163]: i'm not sure if this key is node name or the actual key, as the same key is used at both places.

Comment (gsps1) [inline cdap-app-templates/cdap-etl/cdap-data-pipeline/src/test/java/co/cask/cdap/datapipeline/DataPipelineTest.java:163]: maybe you are using the same key for node name and key ?

**Comment by gsps1** [comment]
the test case looked weird, but it's because WorflowMangaer API is weird. can file a JIRA to improve that. LGTM

**Comment by sagarkapare** [comment]
Filed JIRA https://issues.cask.co/browse/CDAP-6593. Build is green. Merging.

**Pull request merged**